source/index.rst

```diff
@@ -0,0 +1,30 @@
+.. www.mantidproject.org documentation master file, created by
+   sphinx-quickstart on Tue May 25 14:09:03 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+.. toctree::
+   :hidden:
+   :maxdepth: 2
+
+About Mantid
+============
+The Mantid project provides a framework that supports high-performance computing and visualisation of materials science data.
+Mantid has been created to manipulate and analyse neutron scattering and muon spectroscopy data, but could be applied to many other techniques.
+The framework is `open source <https://www.mantidproject.org/Mantid_About.html#License_and_Distribution>`_ and `supported on multiple target platforms (Windows, Linux, Mac). <https://download.mantidproject.org/>`_ It includes and benefits from `many other open source projects <https://github.com/mantidproject/mantid/blob/master/DEPENDENCY_LICENSES.md>`_
+
+Development
+===========
+Mantid is hosted on `GitHub <https://github.com/mantidproject>`_. There are pages on `getting started with development <https://developer.mantidproject.org/GettingStarted.html>`_, `issue tracking <https://developer.mantidproject.org/IssueTracking.html>`_, and more on the development process, tools, testing, etc. on our `developer documentation <https://developer.mantidproject.org/>`_.
+
+Governance
+==========
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse sodales, leo a vestibulum maximus, nibh lorem ullamcorper enim, ac aliquam nisi nibh vel mauris. Sed tincidunt ultrices magna, sit amet dapibus ligula porta non. Sed efficitur, urna a aliquet consectetur, arcu velit blandit urna.
+
+Citing Mantid
+=============
+O. Arnold, et al., Mantid—Data analysis and visualization package for neutron scattering and μSR experiments, Nuclear Instruments and Methods in Physics Research Section A, Volume 764, 11 November 2014, Pages 156-166, http://dx.doi.org/10.1016/j.nima.2014.07.029.
```

Comment:
Can we say something like: "If Mantid contributes to a project that leads to a scientific publication, please acknowledge this work by citing :"

I quite like [matplotlib citing](https://matplotlib.org/stable/citing.html) but I think I'll create that as a separate issue.